src/components/pod/pod.test.js

```diff
@@ -3,7 +3,52 @@ import Pod from "./pod.component";
 import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
 import Content from "../content";
 
-import { podBlock } from "../../../cypress/locators/pod";
+import {
+  podBlock,
+  podComponent,
+  podTitle,
+  podSubTitle,
+  podFooter,
+  podSoftDelete,
+  podContent,
+  podEditContainer,
+  podDelete,
+  podUndo,
+  podEdit,
+} from "../../../cypress/locators/pod";
+
+import { useJQueryCssValueAndAssert } from "../../../cypress/support/component-helper/common-steps";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const PodComponent = ({ ...props }) => {
+  return (
+    <Pod
+      title="Title"
+      subtitle="Subtitle"
+      footer="Footer"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      mb={3}
+      {...props}
+    />
+  );
+};
+
+const SoftDeletePodComponent = ({ ...props }) => {
+  return (
+    <Pod
+      title="Title"
+      subtitle="Subtitle"
+      description="Description"
+      footer="Footer"
+      onUndo={() => {}}
+      {...props}
+    >
+      Soft delete state
+    </Pod>
+  );
+};
 
 const PodWithSoftDelete = () => (
```

Comment:
```suggestion
const SoftDeletePodWithChildren = () => (
```
Appreciate this is not part of your changes, but personally I think now we have `SoftDeletePodComponent` we should rename this slightly to make it clear why this mock component should exist independently to `SoftDeletePodComponent`